mslib/msui/multiple_flightpath_dockwidget.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, parent=None, view=None, listFlightTracks=None,
         self.mscolab_server_url = mscolab_server_url
         self.token = token
         self.ft_settings_dict = self.ui.get_settings()
-        self.color = self.ft_settings_dict["colour_ft_vertices"]
+        self.color = (0, 0, 1, 1)
```

Comment:
I have not understood the issue you encountered, but as I understand it, this dictionary should be intialized properly before this class in the general case...?
Maybe a .get() with this as default value?

Reply:
for a new user there is not that dict defined. Or also not when one deletes the msui-core.conf file. We should not depend on those qsettings created files.